frontend/apps/web/src/pages/system/_name.tsx

```diff
@@ -69,139 +72,14 @@ const RepositoryDetails = () => {
         <Text type={'subtitle l'} color={'gray80'}>
           {totalTests} {t('filecoin.system.totalTests')}
         </Text>
-        <DetailedView>
-          <ReactTooltip
-            effect="solid"
-            getContent={data => {
-              const { functionName, path, repository } = JSON.parse(data) || {}
-              return (
-                <>
-                  <div>
-                    <b>Name</b>: <span>{functionName}</span>
-                  </div>
-
-                  <div>
-                    <b>Path</b>: <span>{path}</span>
-                  </div>
-
-                  <div>
-                    <b>Repository</b>: <span>{repository}</span>
-                  </div>
-                </>
-              )
-            }}
-          />
-
-          <BoxLayout gap={1.5}>
-            <StackLayout gap={1}>
-              <ColumnLayout className={'c-matrix__header'} gap={1}>
-                {testKinds.map(testKind => {
-                  return <Text key={testKind}>{testKind}</Text>
-                })}
-              </ColumnLayout>
-
-              {system.subsystems.map(subsystem => {
-                return (
-                  <ColumnLayout
-                    className={'c-matrix__row'}
-                    gap={1}
-                    key={subsystem.name}
-                  >
-                    <Text>{subsystem.name}</Text>
-
-                    {testKinds.map(testKind => {
-                      const tests = subsystem.tests.filter(
-                        test => test.kind === testKind,
-                      )
-
-                      return (
-                        <MatrixMap
-                          key={testKind}
-                          data={tests}
-                          onClick={() => null}
-                        />
-                      )
-                    })}
-                  </ColumnLayout>
-                )
-              })}
-
-              <Legend>
-                <div>
-                  Missing tests <div />
-                </div>
-                <div>
-                  Passing tests <div />
-                </div>
-                <div>
-                  Failing tests <div />
-                </div>
-              </Legend>
-            </StackLayout>
-          </BoxLayout>
-        </DetailedView>
+        {activeTab === 'overview' ? (
+          <Overview model={model} systemName={params.name} />
+        ) : (
```

Comment:
This can potentially be a glich. For example if it 'overview2' than it will show the DetailedView for a milisec and then after useEffect corrects the link this will show Overview component.